app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CustomersController < ApplicationController
+  before_action :set_customer, only: [:show, :edit, :update]
+
+  def new
+    @customer = Customer.new
+  end
+
+  def create
+    @customer = Customer.new(customer_params)
+
+    if @customer.save
+      flash[:notice] = 'Customer created successful'
+      redirect_to @customer
+    else
+      flash[:alert] = "Customer can't be created"
```

Comment:
I don't understand: why did you change locale to _pt-br_, but you are using _en_ here?